mbedtls/build/crate_id.rs

```diff
@@ -0,0 +1,52 @@
+use std::fmt;
+
+pub struct StableCrateId(u64);
+
+impl StableCrateId {
+    pub fn new(
+        crate_name: &str,
+        is_exe: bool,
+        mut metadata: Vec<String>,
+        cfg_version: String,
+    ) -> StableCrateId {
+        use std::collections::hash_map::DefaultHasher;
+        use std::hash::{Hash, Hasher};
+
+        let mut hasher = DefaultHasher::new();
```

Comment:
As the document of `DefaultHasher`:
```
The internal algorithm is not specified, and so it and its hashes should not be relied upon over releases.
```
Could you update code to explicitly choose hash algo here?

Reply:
Done